resources/links/LinksUpdateCategory.py

```diff
@@ -0,0 +1,37 @@
+from flask_restful import Resource, reqparse
+from flask import jsonify, make_response
+from flask_jwt_extended import jwt_required, get_jwt_identity
+from flasgger import swag_from
+
+from common.DbHandler import DbHandler
+
+
+parser = reqparse.RequestParser(bundle_errors=True)
+parser.add_argument(
```

Comment:
I think you should add 
`, required=True`
otherwise what will happen if somebody doesn't send `new_categories` ?
https://flask-restful.readthedocs.io/en/latest/reqparse.html#error-handling

or maybe it is possible to say we want to update our link and remove all the related categories from it, then what will happen in line 23? `new_categories = args['new_categories']`

Reply:
If you don't send `new_categories`, then all of the categories for that link will be deleted:

![Screenshot_2020-12-01_11-46-41](https://user-images.githubusercontent.com/13691777/100714482-0dedd480-33cb-11eb-974e-bf9ebd32a0a9.png)

![Screenshot_2020-12-01_11-46-52](https://user-images.githubusercontent.com/13691777/100714496-13e3b580-33cb-11eb-9b0d-e2028930f650.png)

Reply:
Nice, thanks :)